### PR TITLE
fixing 4*4 gird layout

### PR DESCRIPTION
**Pull request by subash431**

### CODE REVIEW

**Comment by Ananta580** [comment]
Hi @subash431 , Did you add tailwind right now? As per your commit

**Comment by subash431** [comment]
yes

File: backend/package.json [hunk 1]
```diff
@@ -7,9 +7,5 @@
     "express": "^4.21.1",
     "jsonwebtoken": "^9.0.2",
     "mongoose": "^8.8.3"
-  },
```
Comment (Ananta580) [inline backend/package.json:10]: Why did you removed this, please leve it back there

File: frontend/src/components/Navbar.js [hunk 2]
```diff
@@ -41,16 +41,7 @@ const Navbar = ({ onLogout }) => {
         >
           Gallery
         </Link>
```
Comment (Ananta580) [inline frontend/src/components/Navbar.js:43]: Is this intentional, you removed the search button?

**Review by Ananta580 — APPROVED** [review]

File: frontend/src/components/Navbar.js (hunk 2 above)
Comment (subash431) [reply frontend/src/components/Navbar.js:43]: I added the new search button 
I am going to push

Comment (Ananta580) [reply frontend/src/components/Navbar.js:43]: Okay, Once you push I will review again

**Pull request merged**